### DISCUSSION TITLE
Why not make a PS AI?

### DISCUSSION THREAD

**Discussion by dkqjhoifh** [Acly/krita-ai-diffusion#1257] [Ideas]
For the sake of krita-ai-diffusion, I tried to give up Photoshop. Now, it's been four months and I would like to say, I'm miserable, purely from a usage point of view, krita is very, very bad, both the operation and the various details, it's very bad and mutilated for me, I wouldn't be using krita in my life if it weren't for krita-ai-diffusion, so please don't let this excellent AI plugin become a niche product of Krita, can you make a photoshop-ai-diffusion?
Some Chinese users have started extracting Krita's workflow and trying to make a ps-ai-diffusion, but it's so bad it's almost hard to use (of course, its UI looks pretty good). I don't think anyone is better suited for the job than you! Please consider making ps-ai-diffusion!
This is ps-ai-diffusion made by a Chinese friend!
https://www.bilibili.com/video/BV16PxEeVEfN/?spm_id_from=333.337.search-card.all.click&vd_source=8aaacb87515469df4606a33d29f31ec6

**Comment by dkqjhoifh** [comment]
Of all the AI plugins I use, yours has an overwhelming advantage, and I really don't want to see him being dragged down by Krita, so I took the liberty of making this request@Acly

**Reply by yen-shi-lun** [reply]
When you ask for this feature to be developed, first ask yourself if Acly is sponsoring the author. I really like krita-ai-diffusion using the open source krita+krita ai diffusion.

**Comment by Drakosha405** [comment]
https://github.com/NimaNzrii/comfyui-photoshop 
In it, you can set up the work yourself and get closer to the krita-ai-diffusion workflow. if you are not satisfied with krita.

**Reply by dkqjhoifh** [reply]
I'll give it a try, and artists like me who don't know how to program always struggle with it, T-T

**Comment by Acly** [comment]
I always think this is obvious, but it gets asked every few months.

A PS plugin is basically a rewrite (everything is different) so it will take some months. Maintaining both is even more work and means the plugins will soon lag behind. So you have to ask, why would I do it?

Adobe offers their own AI. Do I want to compete with a billion$ company, _in their own product_, which they have 100% control of? Even if I thought I could, they can just shut down or ban the plugin lol. It's their product.

Only way I would consider it is if someone pays for the development.

**Reply by dkqjhoifh** [reply]
Maybe you can try fundraising?

**Reply by Marksusu** [reply]
> Maybe you can try fundraising?

那还不如在 Shotcut 里做呢，AI 生成的最终目标里，视频是不可能被排除的。

**Comment by zengels** [comment]
Hello,

I wanted to mention the [Auto-Photoshop-StableDiffusion-Plugin](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin). I tried this plugin a month ago, and it was truly impressive, but I’m unsure if it still works, given how tightly Adobe controls their products. What Acly has done is simply incredible. Not only did I discover Krita—a fantastic piece of software—but Acly also made local AI generation accessible and practical for everyone.

I sincerely hope Acly continues developing on Krita and doesn't waste time on Photoshop. I'm incredibly grateful for all this free work! Acly’s contributions have been key in making Krita, along with Krita AI Diffusion, a true open-source alternative to Photoshop and Firefly. How can people not see that? If we want to support Acly, we should spread the word and let people know that there is a real alternative. I had never even heard of Krita before Acly's work.

Perhaps with more awareness, Acly and Krita will attract more people willing to contribute financially to keep this software open-source and free. That’s what open-source is all about—making cutting-edge technology accessible to those who can't afford it, while encouraging those who can to support its continued development.

**Comment by freecoderwaifu** [comment]
Krita is open source so it's easier to support, PS isn't really open, though it does seem to have decent API support for extensions. I wanted a similar plugin for Clip Studio Paint too, but that's also even more closed and it's worse because Celsys is seemingly entirely against unapproved third party extensions.

Coming from CSP, Krita took a lot of getting used to, but ultimately it wasn't that hard and anything that can't be done in Krita or is harder to do I can just go back and forth between programs. You can still go back and forth between PS and Krita too by opening PSD files between them or copy and paste layers (though that might sometimes reduce image quality).

**Comment by yen-shi-lun** [comment]
Help you develop for free and let you sell courses in China?
It supports open source. If you make a ps ai plug-in, is it still called open source?
If you don’t know how to use krita, you don’t need to slander it. Many painters are using krita.

**Reply by dkqjhoifh** [reply]
No, I didn't use AI to make money, I just used Photoshop for ten years, and a lot of instinctive things are engraved in my blood and it's hard to change for a while.

**Reply by Marksusu** [reply]
> No, I didn't use AI to make money, I just used Photoshop for ten years, and a lot of instinctive things are engraved in my blood and it's hard to change for a while.

原本我从 PS 6.0 的时代就开始用它，经历了许多次升级，后来离开工作单位就不再用了。
PS 好是好，就是太贵了，个人用户租软件太不划算。

自从换了 Krita，的确有些多年养成的习惯需要重新去适应，除了动画功能不太行，其它还算够用。

吾等向钱低头，并不会觉得丢脸😂，后来逐渐把 MacOS 和 Win 也抛弃了，无力供养😅😂😂

**Comment by Acly** [comment]
Ok stop squabbling now.
